sdk/identity/identity/test/assets/cert.pem

```diff
@@ -0,0 +1,48 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I wonder if this file is going to upset CredScan

Reply:
A similar cert is being used on other languages. They're not associated with the account, we use a different one to record the tests.